chaos_genius/databases/models/kpi_model.py

```diff
@@ -98,6 +102,16 @@ def as_dict(self):
             "created_at": self.created_at,
         }
 
+
+    @property
+    def multidim_status(self):
```

Comment:
Please remove this as well

Reply:
Any model file should have data/info related to the database schema and meta info only. This must not import any other model for sure and shouldn't have any business logic.

Reply:
do you suggest a fuction placed in some util file ?